package.json

```diff
@@ -62,14 +62,9 @@
     "eslint": "8.17.0",
     "eslint-config-standard": "17.0.0",
     "eslint-config-standard-jsx": "11.0.0",
-    "eslint-plugin-import": "2.26.0",
     "eslint-plugin-jest": "26.5.3",
     "eslint-plugin-jsx-a11y": "6.5.1",
-    "eslint-plugin-n": "15.2.2",
-    "eslint-plugin-node": "11.1.0",
-    "eslint-plugin-promise": "6.0.0",
```

Comment:
should we remove this one ? or actually activate it? for the latter we'll get some errors we have to fix

Reply:
since we didn't use it (at least recently) i would remove them to reduce the deps footprint. some of them even have the same rules. What do you think @phillimorland ?

Reply:
as we have n I think we should remove this one